index.js

```diff
@@ -16,8 +16,18 @@ const StartLedger = typeof process.env.LEDGER === 'undefined' ? 32570 : parseInt
 console.log('Fetch XRPL transactions into Google BigQuery')
 
 const Client = new XrplClient(XRPLNodeUrl)
+
+async function safeHalt() {
+  try {
+    await Client.close()
+  } catch(e) {
+    console.error('ERROR closing connection:', e)
+  } finally {
+    process.exit(1)
+  }
+}
```

Comment:
@WietseWind yes this should be shared code since both the ledger and tx scripts use this function.

but i think that code re-org is required for that, which is outside the scope of this PR. i'm just trying to keep the lights on here and not make anything _worse_ lol